lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -291,4 +300,24 @@ public function generate_styles() {
 
 		return $styles;
 	}
+
+	/**
+	 * Preload webfonts to improve performance
+	 *
+	 * @return void
+	 */
+	public function preload_webfonts() {
+		foreach ( $this->get_fonts() as $font ) {
+			if ( $font['preload'] ) {
+				foreach( $font['src'] as $src ) {
+					$srcAsArray = explode( '.', $src );
+					$file_type = end( $srcAsArray );
```

Comment:
WP Coding Standards nitpick: We should use snake_case instead of camelCase , and align the `=`
```suggestion
					$src_as_array = explode( '.', $src );
					$file_type    = end( $src_as_array );
```